docs/topics/i18n/translation.txt

```diff
@@ -215,7 +215,8 @@ Comparing ``count`` to 1 isn't always the correct rule. This code looks
 sophisticated, but will produce incorrect results for some languages::
 
     from django.utils.translation import ngettext
-    from myapp.models import Report
+
+    from .models import Report
```

Comment:
Leave alone. This looks like a shell session.

Reply:
ok